### ISSUE TITLE
For Yoochoose dataset, why split train-test then get 1/4 and 1/64

### ISSUE THREAD

**Issue by Zziwei** [CRIPAC-DIG/SR-GNN#10]
Hi, 

Thank you for sharing your code. 

I have a question about the data preprocessing for Yoochoose dataset. In the code, you first split the train-test sets based on time, and make sure there is no item in test set that does not appear in training set, then you use the current training data to get most recent 1/4 and 1/64 sessions to be the final training set. By this, the final test set still has items that do not appear in the training set. Is that a problem?

**Comment by SXKDZ** [comment]
Hello, thanks for your interest. We filtered out any items that didn't appear in the training set.
